projects/15_basic_material/build.gradle

```diff
@@ -1,23 +1,13 @@
-task copyTask {
-    copy {
-        from rootProject.ext.assetsPath + '/basic/models'
-        into 'assets/basic/models'
-    }
-    copy {
-        from rootProject.ext.assetsPath + '/basic/shaders/spv'
-        into 'assets/basic/shaders/spv'
-        include '*.spv'
-    }
-    copy {
-        from rootProject.ext.assetsPath + '/common/textures'
-        into 'assets/common/textures'
-    }
-    copy {
-        from rootProject.ext.assetsPath + '/materials/shaders'
-        into 'assets/materials/shaders'
-    }
-    copy {
-        from rootProject.ext.thirdPartyAssetsPath + '/poly_haven'
-        into 'assets/poly_haven'
-    }
-}
+ArrayList projectAssets = [
+    "basic/models",
+    "basic/shaders/spv",
+    "common/textures",
+    "materials/shaders",
```

Comment:
`/spv` at the end? (seems like we just copy additional HLSL files)

Reply:
thanks! fixed